partner_validate_email/README.rst

```diff
@@ -7,14 +7,15 @@ Partner - Validate Email
 
 Version
 =======
-9.0
+12.0
 
 Installation
 ============
 * Required library:
 	* Python lib validate_email <https://pypi.python.org/pypi/validate_email>
     	* `pip install validate_email`
-    	* `pip install pyDNS`
+    * Python lib py3dns <https://pypi.org/project/py3dns>
+    	* `pip install py3dns`
```

Comment:
Can you also create a `requirements.txt` file in this branch, and include these dependencies there?